server/src/models/config.js

```diff
@@ -5,10 +5,21 @@ import mongoose from 'mongoose';
 const Schema = mongoose.Schema;
 const configSchema = new Schema({
   // user: { type: mongoose.Schema.Types.ObjectId, ref: 'User', childPath: 'githubID' },
-  appName: String,
-  port: Number,
-  expressName: String,
-  serverType: String,
+  data: {
+    serverType: String,
+    appName: String,
+    serverSettings: {
+      port: String,
+    },
+    routers: [{
+      id: String,
+      startPoint: String,
+      endPoints: [{}],
```

Comment:
In the state, `endPoints` is not camel-cased. We should maintain consistency and change one or the other.